### PR TITLE
Fixed Bug in Pause/Resume feature

### LINKED ISSUE #66: Fix Bug in Pause/Resume Feature
**Describe the bug**
Right now there's a bug in pause/resume profile feature. Consider you have a profile(Pro A) running, and there's another profile(Pro B) present for other day having starting time between the time of activity of active profile. 
When the start time of profile B becomes equal to the current time, the sharedpreference value stored in the ACTIVE_PROFILE_ID stores id of Profile B (where ideally it should have stored id of Profile B).
When end time of profile A triggers, it does not ENDS because of the change in value of  ACTIVE_PROFILE_ID.

**To Reproduce**

1. Create Profile A (Start: 08:25AM ; End: 08:35AM ; DaySelected: Monday(current day))
2. Create Profile B (Start: 08:27AM ; End: Take any(does not matter) ; DaySelected: Wednesday)
3. For better reference add a Toast displaying value of ACTIVE_PROFILE_ID
4. Observe that at 08:27AM, value stored in ACTIVE_PROFILE_ID changes (from id of A to id of B)

This change in value of ACTIVE_PROFILE_ID will create trouble if you further try to pause/resume profile A (after 8:27AM) 

**Expected behavior**

In the above scenario given since Profile B must be active on Wednesday, so on Moday at 8:27AM, value of ACTIVE_PROFILE_ID  must store id of profile A & not B

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by coder2699**
Fixes #66

### CODE REVIEW

**Comment by coder2699** [comment]
@rob729 I have opened a new PR please check it. There was some problem when I was trying to generate PR from development branch. Sorry for that mess.

File: app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt [hunk 1]
```diff
@@ -172,10 +169,10 @@ class NewProfileFragment : Fragment() {
         }
         val args = arguments?.getParcelable<Profile>("Profile")
         if (args != null) {
-            days = selectedDays.fromJson(args.d, type)
+            days = Utils.daysList(args.d)
             Utils.selectedDays(days, binding.dayPicker)
             binding.userToDoEditText.setText(args.name)
-            if (!args.notes.isBlank()) {
+            if (args.notes != "") {
```
Comment (rob729) [inline app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt]: why change isBlank() to !=""?

Comment (coder2699) [reply app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt]: @rob729 I have made the correction

**Review by rob729 — APPROVED** [review]

**Comment by rob729** [comment]
give @coder2699 5 points

**Pull request merged**